challenges/funding.md

```diff
@@ -0,0 +1,108 @@
+# Funding
+
+Funding RSE can be challenging as we are often split between different forms of funding
```

Comment:
```suggestion
Funding RSE can be challenging as we are often split between different forms of funding.
```
Why is that challenging? Or is it that different funding types bring different challenges, outlined below?